cyclonedx-lib/src/temurin/sbom/TemurinGenSBOM.java

```diff
@@ -176,36 +185,52 @@ static Bom createBom() {
         return bom;
     }
 
-    static void signSBOM(final String keystoreFile, final String keystorePassword, final String privateKey, final String publicKey) {
+    static void signBOM() {
```

Comment:
Keep naming consistent, signSBOM()
Also this needs parameters signSBOM(String jsonFile, String pemFile)